tox.ini

```diff
@@ -40,11 +40,11 @@ commands = black .
 [testenv:benchmarks]
 description = Dry run the benchmarks to check for errors
 basepython = python3
-deps = asv
+deps = git+https://github.com/airspeed-velocity/asv.git@master#egg=asv
 changedir = benchmarks
 commands =
 		 asv machine --yes
-		 asv dev
+		 asv run -e --quick --strict --show-stderr --python=same
```

Comment:
Is `run --quick` basically the new `dev`? So a "faster" test-run of the benchmarks?

Reply:
`asv dev` is equivalent to `asv run -e --quick --show-stderr --python=same` by additionally adding `--dry-run` and `--strict` we can a) avoid saving any results to disk and b) ensure a non-zero return value is given on failure.

Reply:
Thanks, good to know :)